configure

```diff
@@ -676,32 +677,33 @@ for lib in $generated_META $lbytes; do
         -e "s|%%interfaces%%|${if}|g" \
         -e "s|%%findlib_version%%|${version}|g" \
         -e "s|%%natdynlink%%|${natdynlink}|g" \
-        -e "s|%%dynlink_dir%%|${dynlink_dir}|g" \
-        -e "s|%%unix_dir%%|${unix_dir}|g" \
-        -e "s|%%str_dir%%|${str_dir}|g" \
-        site-lib-src/$lib/META.in > site-lib-src/$lib/META
+        -e "s|%%dynlink_dir%%||g" \
+        -e "s|%%unix_dir%%||g" \
+        -e "s|%%str_dir%%||g" \
+        site-lib-src/"$lib"/META.in > site-lib-src/"$lib"/META
 
     echo "Configuration for $lib written to site-lib-src/$lib/META"
 done
 
-for part in `cd src; echo *`; do
-    if [ -f "src/$part/META.in" ]; then
-	sed -e "s/@VERSION@/$version/g" \
-            -e "s/@REQUIRES@/${req_bytes}/g" \
-            src/$part/META.in >src/$part/META
-    fi
-done
+tmp="$(mktemp)"
+find src ! -name "$(printf "*\n*")" -name 'META.in' -type f > "$tmp"
```

Comment:
the part `-name "$(printf ...)"` looks questionable. What is the intended meaning?

Reply:
it is `! -name $(printf ...)` to skip over names that have newlines. The issue is [described as SC2044](https://www.shellcheck.net/wiki/SC2044) and a better readable solution is possible but only by using bashisms. Not that we expect files with newlines here…

The other possible portable POSIX solution is to use `find`s `-exec`, but I am not sure how readable a longer `exec` command is.

Reply:
I've ended up rewriting this using `-exec` (as described in the wiki above) as this gets rid of the temporary files and with the `${part%.in}` variable substitution form it is halfway readable.